docs/import/list_ee_assets.js

```diff
@@ -45,7 +45,8 @@ function getStatesAssetsFromEe(states) {
  * of asset path to type for the given disaster.
  */
 function getDisasterAssetsFromEe(disaster) {
-  return listEeAssets(eeLegacyPathPrefix + disaster);
+  return listEeAssets(eeLegacyPathPrefix + disaster)
```

Comment:
We're just returning the asset paths now? Does that mean disasterAssets isn't storing type information anymore? Does this break createDisasterAssetPickers?

Reply:
Look at latest revision :)